src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -455,6 +463,9 @@
     <None Update="ProjectSystem\Rules\AdditionalDesignTimeBuildInput.xaml">
       <Generator>MSBuild:Compile</Generator>
     </None>
+    <None Update="ProjectSystem\Rules\PropertyPages\BuildPropertyPage.xaml">
+      <Generator>MSBuild:Compile</Generator>
+    </None>
```

Comment:
I think the project system added this for you automatically, but you shouldn't need it.

Reply:
We could/should add a `<None Remove />` glob to stop this from happening

Reply:
I removed it and seems like it doesn't generates it anymore, but will consider it if it comes back.